incubator/kubeless/templates/kafka-svc.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: kafka
+  namespace: kubeless
+  labels:
```

Comment:
Label `kubeless` (or rather `component`?) should be added here.

Reply:
added standard labels